flight/Waypoint/Move_To_Test.py

```diff
@@ -0,0 +1,89 @@
+"""
+Main driver code for moving drone to each waypoint
+"""
+
+import asyncio
+import json
+import argparse
+import goto
+from mavsdk import System
+import mavsdk as sdk
+import logging
+import math
+import typing
+from typing import Dict, List
+import sys
+
+
+async def run() -> None:
+    """
+    This function is just a driver to test the goto function and runs through the
+    entire waypoint section of the SUAS competition
+    """
+    # Put all latitudes, longitudes and altitudes into seperate arrays
+    lats: List[float] = [37.9008502, 37.9008129, 37.8964543]
+    longs: List[float] = [-91.6615228, -91.6615335, -91.6570381]
+    altitudes: List[float] = [100, 100, 100]
```

Comment:
Instead of using a list here, just make altitude a single integer, and use it for the goto function. Or, better yet, just pass 100 or 75 in the argument list.